src/Commands/Migrate.php

```diff
@@ -42,17 +46,24 @@ public function handle(): int
             return 1;
         }
 
-        tenancy()->runForMultiple($this->getTenants(), function ($tenant) {
-            $this->line("Tenant: {$tenant->getTenantKey()}");
+        try {
+            tenancy()->runForMultiple($this->option('tenants'), function ($tenant) {
+                $this->line("Tenant: {$tenant->getTenantKey()}");
 
-            event(new MigratingDatabase($tenant));
+                event(new MigratingDatabase($tenant));
+                // Migrate
+                parent::handle();
 
-            // Migrate
-            parent::handle();
+                event(new DatabaseMigrated($tenant));
+            });
 
-            event(new DatabaseMigrated($tenant));
-        });
+            return 0;
+        } catch (TenantDatabaseDoesNotExistException|QueryException $th) {
+            if (! $this->option('skip-failing')) {
+                throw $th;
+            }
 
-        return 0;
+            return 1;
```

Comment:
Looking at the TenantDump command, I see that the `Command::SUCCESS` and `Command::FAILURE` constants exist. Maybe we could use these instead of just 0 and 1? That would be prettier imo @stancl

Reply:
`0` and `1` return codes in commands are super standard, even Laravel uses them, so I think we'll stick with that. I didn't know about the constants so that's cool that they're available though in this case we'll stick with the simple solution.

Reply:
Okay, I see